src/app/features/tech-record/components/tech-amend-reason/tech-amend-reason.component.ts

```diff
@@ -23,14 +23,13 @@ export class TechAmendReasonComponent {
 
     this.form = new CustomFormGroup(
       { name: 'reasonForAmend', type: FormNodeTypes.GROUP },
-      { reason: new CustomFormControl({ name: 'reason', type: FormNodeTypes.CONTROL }, 2, [Validators.required]) }
+      { reason: new CustomFormControl({ name: 'reason', type: FormNodeTypes.CONTROL }, 'correcting-an-error', [Validators.required]) }
```

Comment:
If this is to match new enum, would be better to use the enum here when initialising the control value.